tests/mock_crust_tests.rs

```diff
@@ -40,6 +40,10 @@
 extern crate itertools;
 #[macro_use]
 extern crate log;
+#[cfg_attr(feature="clippy", allow(useless_attribute))]
+#[allow(unused_extern_crates)]
+#[macro_use]
+extern crate maidsafe_utilities;
```

Comment:
Um, not sure.  I'd have thought `#[cfg(test)]` would be true for all files inside the /tests folder?  And we already have [`use-mock-crust` set module wide](https://github.com/maidsafe/routing/pull/1254/files#diff-08fe2788cc792d4f0a04808a8c92da53R18), so `#cfg[cfg_attr(feature="use-mock-crust")]` will also always be true here?

Basically, I'd like to just have
```rust
#[macro_use]
extern crate maidsafe_utilities;
```
but that yields a compiler warning about an unused extern crate since we're only using a macro from it.  That forces the use of `#[allow(unused_extern_crates)]`, but clippy flags that as a "useless attribute", hence the need to disable the clippy check for that import line.